### ISSUE TITLE
CLI set method is not working as documented

### ISSUE THREAD

**Issue by ShivKJ** [theskumar/python-dotenv#345]
Hi,

Thanks for creating such a useful library in Python.

I was exploring the full potential of the library and that is when I came across its [CLI doc](https://github.com/theskumar/python-dotenv#command-line-interface). 

I am currently using version `0.18.0` and there is a `dotenv set` method that is not working as documented.

When I run,
```bash
dotenv set USER=foo
```
I get the following error,
```
Usage: dotenv set [OPTIONS] KEY VALUE
Try 'dotenv set --help' for help.

Error: Missing argument 'VALUE'.
```

However when I run the following,
```bash
dotenv set USER foo
```
then it correctly updates the `.env`  file in current directory.

In other words, **NOT using "=" works**

**Comment by bbc2** [comment]
That's right.  Thanks for reporting!
